### PR TITLE
feat(twenty-website): add stargazer count

### PR DESCRIPTION
**Pull request by gazjones00**
Addresses #4007

### CODE REVIEW

**Comment by FelixMalfait** [comment]
Oh man I'm sorry. There was an ongoing PR for this which I just merged. 
I apologize, I should have noticed and linked the original issue :( 
Let me know if you work on anything else, happy to support you and avoid wasting your time like this again! Thanks!!!

File: packages/twenty-website/src/app/_server-utils/fetch-stargazers-count.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export async function fetchStargazerCount(): Promise<string | null> {
+  /**
+   * Fetch the stargazers count from the GitHub API and revaildate every 1 hour
+   * to avoid hitting the rate limit
+   */
+  try {
+    const res = await fetch('https://api.github.com/repos/twentyhq/twenty', {
+      next: { revalidate: 3600 },
```
Comment (FelixMalfait) [inline packages/twenty-website/src/app/_server-utils/fetch-stargazers-count.ts:8]: Didn't know about this revalidate param, nice. We already had a function/cron to sync Github data to the Postgres db so we went this path - but yours is elegant and straightforward, I like it

Comment (gazjones00) [reply packages/twenty-website/src/app/_server-utils/fetch-stargazers-count.ts:8]: Yeah it saves on having to store simple data like this in a database unnecessarily by just storing it in memory, see reference docs here - https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-server-with-fetch

**Comment by gazjones00** [comment]
> Oh man I'm sorry. There was an ongoing PR for this which I just merged. 
> 
> I apologize, I should have noticed and linked the original issue :( 
> 
> Let me know if you work on anything else, happy to support you and avoid wasting your time like this again! Thanks!!!

Sure thing no worries, I was just looking for an open source project to have some involvement in during my downtime. I like to easy in with a smaller task for to get a feel for the codebase :)

**Comment by FelixMalfait** [comment]
Thanks! Feel free to look at issues tagged "good first issues" (easier) or "good for experienced contributors" (hard). I'll try to do a cleanup of those issues today

**Pull request closed without merging**